docker/mongodb-kubernetes-tests/tests/multicluster/multi_cluster_scale_up_cluster.py

```diff
@@ -139,3 +159,59 @@ def test_ops_manager_has_been_updated_correctly_after_scaling():
 def test_replica_set_is_reachable(mongodb_multi: MongoDBMulti, ca_path: str):
     tester = mongodb_multi.tester()
     tester.assert_connectivity(opts=[with_tls(use_tls=True, ca_path=ca_path)])
+
+
+# From here on, the tests are for verifying that we can change the project of the MongoDBMulti resource even with
```

Comment:
nit: I find this comment a bit unnecessary, but you don't need to remove it. 
It would be better to wrap such a set of test functions into a class and name it properly, e.g. `TestNonSequentialMemberIdsInReplicaSet` or sth like that.